mkosi/manifest.py

```diff
@@ -214,6 +216,17 @@ def record_pkg_packages(self) -> None:
                 self.source_packages[source] = source_package
             source_package.add(package)
 
+    def record_extension_release(self) -> None:
+        if self.context.config.output_format not in (OutputFormat.sysext, OutputFormat.confext):
+            return
+
+        with complete_step("Recording sysext / confext information in manifest…"):
```

Comment:
No need for `sysext / confext` here, you already calculate which one just below, just use that in the step message instead of listing both.

Reply:
This is still missing?

Reply:
Should be there now .`with complete_step(f"Recording {type} information in manifest…"):